src/dmd/expressionsem.d

```diff
@@ -94,10 +94,16 @@ enum LOGSEMANTIC = false;
  */
 bool expressionsToString(ref OutBuffer buf, Scope* sc, Expressions* exps)
 {
-    if (!exps)
+    return expressionsToString(buf, sc, exps.opSlice);
+}
+
+/// ditto
+bool expressionsToString(ref OutBuffer buf, Scope* sc, Expression[] exps)
```

Comment:
added a slice overload so I can call this with a single expression using `(&expr)[0 .. 1]`